### PR TITLE
Backport changes made in kawennonnis here

### PR DESCRIPTION
**Pull request by joanise**
This PR will let us redo the kawennonnis history by merging from here instead of duplicating commits there.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,11 @@
-
 # Created by https://www.gitignore.io/api/macos,python
 # Edit at https://www.gitignore.io/?templates=macos,python
 
 ### Custom ###
 docker-stack.yml
+src/wordweaver/data/*/conjugations.json
```
Comment (roedoejet) [inline .gitignore:6]: I thought we weren't using uncompressed conjugations anymore?

File: docker-compose.yml [hunk 2]
```diff
@@ -8,13 +8,13 @@ services:
...
-      command: 'bash -c "wordweaver validate && wordweaver compress && /start-reload.sh"'
+      command: 'bash -c "/start-reload.sh"'
       env_file:
          - env-backend.env
       environment:
          - SERVER_HOST=http://${DOMAIN}
          - SERVER_NAME=${DOMAIN}
       ports:
-         - 80:8080
+         - 80:80
```
Comment (roedoejet) [inline docker-compose.yml:18]: Why no keep the port 8080? I like to keep my docker ports a little different partially for debugging

**Review by roedoejet — COMMENTED** [review]
A couple small questions, looks good though, thanks @joanise !

File: .gitignore (hunk 1 above)
Comment (joanise) [reply .gitignore:6]: That's why I want to .gitignore it now.

File: docker-compose.yml (hunk 2 above)
Comment (joanise) [reply docker-compose.yml:18]: That's a change you did on kawennonnis_api_v2 that I'm just backporting.

Comment (joanise) [reply docker-compose.yml:18]: You had modified `env-backend.env` and in order to work, `docker-compose.yml` has to use the same port.

**Pull request merged**